.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Delete this sentence after you read the "Reporting bugs" paragraph from the
+"guidelines for contributing" link above.
+
+If you are reporting a bug, please try to fill in the following, otherwise
+remove it along with this sentence.
+
+| Question       | Answer
+|----------------|-------------------------------------------------------------
+| Bundle version | composer show --installed|grep sonata-project/admin-bundle
```

Comment:
> Should have back-ticks on command lines.

Really ? This is not supposed to be converted to html, it will only appear in a textarea, won't it? The command lines are meant to be copy pasted

> Spaces between | and commands would be easier to read. 

Ok